testnet/stacks-node/src/nakamoto_node/miner.rs

```diff
@@ -291,25 +287,26 @@ impl BlockMinerThread {
             ));
         };
 
-        let mut chain_state = neon_node::open_chainstate_with_faults(&self.config)
+        let chain_state = neon_node::open_chainstate_with_faults(&self.config)
             .expect("FATAL: could not open chainstate DB");
-        let sortition_handle = sort_db.index_handle_at_tip();
-        let Ok(aggregate_public_key) = NakamotoChainState::get_aggregate_public_key(
-            &mut chain_state,
-            &sort_db,
-            &sortition_handle,
-            &new_block,
-        ) else {
-            return Err(NakamotoNodeError::SigningCoordinatorFailure(
-                "Failed to obtain the active aggregate public key. Cannot mine!".into(),
-            ));
+
+        // NOTE: this is a placeholder until the API can be fixed
+        let aggregate_public_key = {
+            let key_bytes = [
+                0x03, 0xd3, 0xe1, 0x5a, 0x36, 0xf3, 0x2a, 0x9e, 0x71, 0x31, 0x7f, 0xcb, 0x4a, 0x20,
+                0x1b, 0x0c, 0x08, 0xb3, 0xbc, 0xfb, 0xdc, 0x8a, 0xee, 0x2e, 0xe4, 0xd2, 0x69, 0x23,
+                0x00, 0x06, 0xb1, 0xa0, 0xcb,
+            ];
+            let ecdsa_pk = ecdsa::PublicKey::try_from(key_bytes.as_slice()).unwrap();
+            Point::try_from(&Compressed::from(ecdsa_pk.to_bytes())).unwrap()
         };
```

Comment:
Why this instead of `Point::new()`?

Reply:
Because I didn't know `Point::new()` was a thing :)  I'll update